### ISSUE TITLE
How do you suggest setting / overriding triggers in a sublime.project file?

### ISSUE THREAD

**Issue by mrbinky3000** [sagold/FuzzyFilePath#26]
I understand that you can override the default settings by creating user settings.  Will setting scope triggers in a sublime.project file take precedence over both the default and user settings?  If not, can you add this feature?

**Comment by sagold** [comment]
Yes that is the case. Settings are merged in the following order: default -> user -> project-settings. Where project-settings comes last. If you open *Edit Project* you should see an entry

```js
{
	"folders":
	[
		{
			"path": "."
		}
	],
	"settings":
	{
		"FuzzyFilePath":
		{
                     // override FFP settings here 
		}
	}
}
```

Remember, the `scopes`-Array will currently be replace. Thus you have to copy any triggers you would like to reuse.

**Comment by sagold** [comment]
With added support of multiple folders (87e56120295d11e27a2ac2545645f4dba59a32fd), each folder may be configured independently:
